### PR TITLE
Target Android SDK 28

### PR DESCRIPTION
**Pull request by sambarboza**
# Description 
- Upgrade Android target SDK to API Level 28 (Android 9)

# Changes
- Removed `UpgradeWalletService`.
- Added adaptive icons.
- Creating notification channels on App startup, required for apps targeting SDK 26 or later.
- Promoting `BlockchainServiceImpl` to foreground service when synchronizing the blockchain, to this purpose an ongoing notification will be shown during a blockchain sync.
- Starting `InactivityNotificationService` as a foreground service.
- Created class `BlockchainStateUtils` to reuse code that calculates status of blockchain sync.

### CODE REVIEW

File: wallet/AndroidManifest.xml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="de.schildbach.wallet_test"
     android:installLocation="internalOnly"
-    android:versionCode="60106"
-    android:versionName="6.1.2">
+    android:versionCode="60200"
```
Comment (HashEngineering) [inline wallet/AndroidManifest.xml]: I like keep version bumps in separate commits so its easy to tell from the history what commits done in each version.
version-commit

**Comment by HashEngineering** [comment]
https://github.com/bitcoin-wallet/bitcoin-wallet/commit/b8c8f03bfb9eb125a3d719b66bbea4e6457714ed#diff-c83987f301cf9a0238316e8e251542b8

In Bitcoin Wallet, wallet/project.properties was also modified, do we need to do that too?

**Comment by HashEngineering** [comment]
In my review, I will compare to Bitcoin Wallet:
https://github.com/bitcoin-wallet/bitcoin-wallet/commit/b8c8f03bfb9eb125a3d719b66bbea4e6457714ed
https://github.com/bitcoin-wallet/bitcoin-wallet/commit/ae1a10b1dc0767727169600732b41b0172b72473
https://github.com/bitcoin-wallet/bitcoin-wallet/commit/be80d62ec8ee9dd14fe5fbfbdf12538a3a2fb84c
https://github.com/bitcoin-wallet/bitcoin-wallet/commit/af752e8213a45ab83d190f2460ac41643a867edf

File: wallet/AndroidManifest.xml (hunk 1 above)
Comment (sambarboza) [reply wallet/AndroidManifest.xml]: Commit #1b36c77 reverts version bump and fixes a crash when service is started by alarm.
